src/Microsoft.PowerShell.Commands.Management/commands/management/CombinePathCommand.cs

```diff
@@ -33,6 +34,7 @@ public class JoinPathCommand : CoreCommandWithCredentialsBase
         [Parameter(Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true)]
         [AllowNull]
         [AllowEmptyString]
+        [SupportsWildcards]
```

Comment:
Ok here is where I wondered about the validity of `[SupportsWildcards]`:

```
> join-path -Path .\PowerShell* -ChildPath *-*
.\PowerShell*\*-*
```

So it looks like `Join-Path` does not support wildcards.

In which case, I feel like the only way to guarantee the correctness of this PR is to test all the labelled parameters in question or read the code for something like `GetResolvedProviderPathFromPSPath()`